### ISSUE TITLE
Persistent display problem

### ISSUE THREAD

**Issue by ghost** [zach-morris/plugin.program.iagl#234]
Hello,
I am a big fan of this application which I use under via RPI4 under Libreelec Kodi Matrix. I have a problem, I accidentally pressed a button on my controller (I can't remember which one) and the resolution when launching video games goes to ultra low resolution. The problem is that as soon as I want to exit the app (start + back) this bug also spreads to Kodi.

I tried everything: reset the app, delete the app, empty the kodi cache, reset the controllers etc ...
I'm afraid I will have to erase my entire Kodi to put the IAGL version back correctly.
I don't see if there is another possibility to reset the resolution of the games

Thank you

**Comment by zach-morris** [comment]
IAGL doesn't do anything with regards to resolution.  It just launches the game.  The resolution switching likely happens in Retroarch, so I'd launch Retroarch and go to Video Settings and update your settings there.

**Comment by kb173** [comment]
I stumbled upon the same problem. Specifically, this is caused by setting the resolution scaling mode to something along the lines of "use original resolution" in the settings menu of a game's pause UI.

Managed to fix it by resetting Kodi's GUI settings: `rm /storage/.kodi/userdata/guisettings.xml`

I'm not sure where that settings UI with the broken configuration comes from. I'm not using Retroarch, so maybe that's just Kodi's own UI for games? That would be consistent with the fact that the problem applied to all emulators, and resetting various addon configs did not help.
